src/redux/reducers/samples/samplesFileUpdate.js

```diff
@@ -20,8 +20,7 @@ const samplesFileUpdate = (state, action) => {
 
   newFile.lastModified = lastModified;
 
-  const newFileNames = new Set([...state[sampleUuid].fileNames, fileName]);
-
+  const newFileNames = Array.from(new Set([...state[sampleUuid].fileNames, fileName]));
```

Comment:
In the initial state fileNames is a set, why are we doing this change?

Reply:
The API expects an array not a set and I was getting errors.